representatives/contrib/francedata/variants.py

```diff
@@ -110,6 +151,31 @@ def _get_an_comittees(json):
     return comittees
 
 
+def _get_an_delegations(json):
+    delegations = []
+    start = json['mandat_debut']
+
+    for g in ([i['responsabilite'] for i in json['responsabilites']] +
+            [j['responsabilite'] for j in json['groupes_parlementaires']]):
```

Comment:
If I remember my scoping lessons correctly, there's no need to name the variable j instead of i here.